plugin/pluginenv/environment_test.go

```diff
@@ -152,13 +149,17 @@ func TestEnvironment(t *testing.T) {
 
 	assert.NoError(t, env.ActivatePlugin("foo"))
 	assert.Equal(t, env.ActivePluginIds(), []string{"foo"})
+
+	assert.Equal(t, env.SearchPath(), dir)
+	assert.Equal(t, env.WebappPath(), webappDir)
+
 	assert.Empty(t, env.Shutdown())
 }
 
 func TestEnvironment_DuplicatePluginError(t *testing.T) {
 	dir := initTmpDir(t, map[string]string{
-		"foo/plugin.json":  `{"id": "foo"}`,
-		"foo2/plugin.json": `{"id": "foo"}`,
+		"foo/plugin.json":  `{"id": "foo", "backend": {}}`,
+		"foo2/plugin.json": `{"id": "foo", "backend": {}}`,
```

Comment:
I feel like duplicate ids should fail regardless of whether the "backend" key was present.

Reply:
Still want either rationale or a change here.